components/eam/cime_config/config_component.xml

```diff
@@ -68,12 +68,14 @@
       <value compset="_EAM%ADIAB"      >-phys adiabatic</value>
 
       <!-- Multiscale modeling framework (MMF) -->
-      <value compset="_EAM%.*MMF"      >-use_MMF -crm samxx -nlev 60 -crm_nz 50</value>
-      <value compset="_EAM%.*MMF"      >-crm_dt 10 -crm_dx 2000 -crm_nx 64 -crm_ny 1 </value>
-      <value compset="_EAM%.*MMF"      >-crm_nx_rad 4 -crm_ny_rad 1 -rad rrtmgp -rrtmgpxx</value>
+      <value compset="_EAM%.*MMF1"     >-crm samxx -crm_dt 10 </value>
+      <value compset="_EAM%.*MMF2_"    >-crm pam -pam_dycor spam  -crm_dt 4 </value>
+      <value compset="_EAM%.*MMF2-AWFL">-crm pam -pam_dycor awfl  -crm_dt 4 </value>
+      <value compset="_EAM%.*MMF"      >-use_MMF -nlev 60 -crm_nz 50</value>
+      <value compset="_EAM%.*MMF"      >-crm_dx 2000 -crm_nx 64 -crm_ny 1 </value>
       <value compset="_EAM%.*MMF1"     >-MMF_microphysics_scheme sam1mom -chem none</value>
-      <value compset="_EAM%.*MMF"      >-use_MMF_VT</value>   <!-- CRM variance transport -->
```

Comment:
do use_MMF_VT and use_MMF_ESMT need to be set in the cam_config_option additionally when using MMF1?

Reply:
Good catch. It actually doesn't matter right now since it's not implemented in PAM yet, but it's worth disabling it for PAM cases so I'll update this line.

Reply:
FYI re-enabled VT and ESMT only for MMF1/SAM option